gravitee-apim-console-webui/src/management/settings/shared-policy-groups/shared-policy-group/shared-policy-group-history/shared-policy-group-history.component.ts

```diff
@@ -196,6 +196,7 @@ export class SharedPolicyGroupHistoryComponent implements OnInit {
       .open<HistoryStudioDialogComponent, HistoryStudioDialogData, HistoryStudioDialogResult>(HistoryStudioDialogComponent, {
         data: {
           sharedPolicyGroup,
+          isReadOnly: sharedPolicyGroup.originContext?.origin === 'KUBERNETES',
```

Comment:
do we have also an issue here if i' haven't SPG Update permission ? 

do we miss this `|| !this.permissionService.hasAnyMatching(['environment-shared_policy_group-u'])` no ?